source/Engine.h

```diff
@@ -127,6 +130,9 @@ class Engine {
 
 	void DoGrudge(const std::shared_ptr<Ship> &target, const Government *attacker);
 
+	unsigned FleetPlacementLimit(const LimitedEvents<Fleet> &fleet, unsigned frames, bool requireGovernment);
```

Comment:
in my personal style guide, that has nothing to do with ES, I try to avoid using just unsigned and prefer uint32_t.
But that's just my personal opinion

Reply:
When there is no need for a specific type size, I prefer not to give one.